hydra-node/hydra-node.cabal

```diff
@@ -70,6 +70,7 @@ common project-config
     -Wincomplete-uni-patterns
     -Wpartial-fields
     -Wredundant-constraints
+    -Wunused-packages
```

Comment:
I thought we said we wanted to remove that one because it was making ghci buggy :thinking: ?

Reply:
I will removed it, added it because I wanted to detect unused packages...